.travis.yml

```diff
@@ -5,18 +5,29 @@ python:
   - "2.7"
   - "3.3"
   - "3.4"
-  - "3.5"  
+  - "3.5"
 before_install:
   - wget 'http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh' -O miniconda.sh
   - chmod +x miniconda.sh
   - ./miniconda.sh -b
   - export PATH=/home/travis/miniconda2/bin:$PATH
   - conda update --yes conda
 install:
-  - conda create --yes -n gensim-test python=$TRAVIS_PYTHON_VERSION pip atlas numpy scipy
+  - conda create --yes -n gensim-test python=$TRAVIS_PYTHON_VERSION pip atlas numpy scipy matplotlib
   - source activate gensim-test
   - pip install pyemd
   - pip install annoy
   - pip install testfixtures
+  - pip install --user travis-sphinx
+  - pip install sphinx-gallery
+  - pip install Pillow
   - python setup.py install
-script: python setup.py test
+script:
+  - cd docs/src
+  - make html
+  - travis-sphinx --source=./ --branches=develop,cidocs -n build
+  - cd ../..
+  # - python setup.py test
+after_success:
+  - cd docs/src
+  - travis-sphinx --branches=develop,cidocs -n deploy
```

Comment:
The cidocs here is for building the github.io page for my branch. TODO: remove it.